mappings/net/minecraft/entity/vehicle/StorageMinecartEntity.mapping

```diff
@@ -8,11 +8,7 @@ CLASS net/minecraft/class_1693 net/minecraft/entity/vehicle/StorageMinecartEntit
 		ARG 4 y
 		ARG 6 z
 		ARG 8 world
-	METHOD <init> (Lnet/minecraft/class_1299;Lnet/minecraft/class_1937;)V
-		ARG 1 type
-	METHOD createMenu (ILnet/minecraft/class_1661;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1703;
-		ARG 1 syncId
-	METHOD method_17357 createMenu (ILnet/minecraft/class_1661;)Lnet/minecraft/class_1703;
+	METHOD method_17357 getMenu (ILnet/minecraft/class_1661;)Lnet/minecraft/class_1703;
```

Comment:
so `getMenu` or `createMenu`?

Reply:
It should be `createMenu`, I think. I'm pretty sure there was a `getContainer` there when I was merging.